tvm_executor/src/transaction_executor.rs

```diff
@@ -298,7 +305,40 @@ pub trait TransactionExecutor {
             fee.add(due_payment)?;
             acc.set_due_payment(None);
         }
-
+        if (tr.now() < acc.last_paid() + STORAGE_FEE_COOLER_TIME) && (is_same_thread_id) {
+            acc_balance.grams.add(&fee)?;
```

Comment:
Do we subtract fee and then add it back? It seems to be bad practice, because first sub can stop execution